app/utils/answersHelpers/consignment/HouseConsignmentAnswersHelper.scala

```diff
@@ -90,11 +90,11 @@ class HouseConsignmentAnswersHelper(
             helper.grossWeightRow,
             helper.netWeightRow
           ).flatten,
-          children = Seq(additionalReferenceSection(houseConsignmentIndex, itemIndex))
+          children = Seq(additionalReferenceSection(itemIndex))
         )
     }
 
-  def additionalReferenceSection(houseConsignmentIndex: Index, itemIndex: Index): Section = {
+  def additionalReferenceSection(itemIndex: Index): Section = {
```

Comment:
Might just be worth moving this logic up into the `children = `. Means we don't need to re-instantiate the helper each time